selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -1068,4 +1072,11 @@ public void waitAllItemsIsSelected(List<String> paths) {
   public void clickOnMaximizeButton() {
     seleniumWebDriverHelper.waitAndClick(By.xpath(Locators.MAXIMIZE_BUTTON_XPATH));
   }
+
+  // Wait for project has PROJECT_FOLDER status
```

Comment:
`Wait for project has PROJECT_FOLDER status` > `Wait project is completely initialized`